# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../madwifi/no-depmod.patch
# Copyright (C) 2006 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-07-14 01:15:56.000000000 -0400
+++ ./Makefile	2006-09-10 12:00:40.000000000 -0400
@@ -83,16 +83,9 @@
 install: install-modules install-tools
 
 install-modules:
-	@# check if there are modules left from an old installation
-	@# might cause make to abort the build
-	sh scripts/find-madwifi-modules.sh $(KERNELRELEASE) $(DESTDIR)
-
 	for i in $(DIRS_MODULES); do \
 		$(MAKE) -C $$i install || exit 1; \
 	done
-ifeq ($(DESTDIR),)
-	(export KMODPATH=$(KMODPATH); /sbin/depmod -ae)
-endif
 
 install-tools:
 	$(MAKE) -C $(TOOLS) install || exit 1
